src/test/java/com/superdevs/HealthOMeter/service/calculatorServiceTests/CalculatorServiceTestsForBMI.java

```diff
@@ -0,0 +1,99 @@
+package com.superdevs.HealthOMeter.service.calculatorServiceTests;
+
+import com.superdevs.HealthOMeter.calculators.BMICalculator;
+import com.superdevs.HealthOMeter.calculators.RFMCalculator;
+import com.superdevs.HealthOMeter.calculators.WHRCalculator;
+import com.superdevs.HealthOMeter.service.CalculatorService;
+import com.superdevs.HealthOMeter.service.managers.CalculatorManager;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class CalculatorServiceTestsForBMI {
```

Comment:
I have got many doubts.
First - All tests are failing. I have downloaded this code to me, compiled all data, and tried to run them. All I received were nested exceptions.
Secondly - Shouldn't the tests tree look the same as does the main project tree? It should look like 1:1. If we are testing CalculatorService.java it should be in the same package as the original. Furthermore, it should verify all methods from the main branch tree. Splitting this into smaller pieces can cause chaos.

Reply:
I agree with @Sephirot92 we don't need to create beans manually. 

We should use: 

@Autowired
CalculatorService calculatorService;